assets/js/theme/search.js

```diff
@@ -201,6 +201,10 @@ export default class Search extends CatalogPage {
                 $searchForm.append(input);
             }
         });
+
+        setTimeout(() => {
+            $('[data-search-aria-message]').removeClass('u-hidden');
+        }, 100);
```

Comment:
Why do we need to remove u-hidden class after timeout? Could we just remove u-hidden class from quickSearchResultsAriaMessage element?

Reply:
We need to update this block after every search request for notification to be read by screen reader. . So firstly we need to hide block and then show it. setTimeout will help to reinit notification node and it will be announced. Timeout in debounce, so it will not affect performance